src/mixxxmainwindow.h

```diff
@@ -148,9 +148,9 @@ class MixxxMainWindow : public QMainWindow {
     std::unique_ptr<mixxx::LibraryExporter> m_pLibraryExporter;
 #endif
 
-    mixxx::TooltipsPreference m_toolTipsCfg;
+    mixxx::Tooltips m_toolTipsCfg;
 
-    mixxx::ScreenSaverPreference m_inhibitScreensaver;
+    mixxx::ScreenSaver m_inhibitScreensaver;
```

Comment:
this looks a little short now, wdyt about reintroducing the preferences namespace again? Adding it to the enum name looks a little clunky.

Reply:
I agree, mixxx:screensaver would imply a subsystem about screensaver handling.  something like mixxx::preferences::screensaver make sense